src/django_ixpmgr/__init__.py

```
@@ -0,0 +1,6 @@
+from .settings import VERSIONED_APP
+
+if VERSIONED_APP == "v5":
+    from django_ixpmgr.v5 import IxpManagerAppConfig
+elif VERSIONED_APP == "v3":
+    from djnago_ixpmgr.v3 import IxpManagerAppConfig
```

`djnago`

we should probably add a unit test or two for v3 models

---

Maybe raise an error at fallthrough here in addition to settings.py

---

Good call, added raise on unknown and fixed typos.